geotiepoints/interpolator.py

```diff
@@ -279,38 +275,29 @@ def interpolate_dask(self, fine_points, method, chunks):
         """Interpolate (lazily) to a dask array."""
         from dask.base import tokenize
         import dask.array as da
+        from dask.array.core import normalize_chunks
         v_fine_points, h_fine_points = fine_points
         shape = len(v_fine_points), len(h_fine_points)
 
-        try:
-            v_chunk_size, h_chunk_size = chunks
-        except TypeError:
-            v_chunk_size, h_chunk_size = chunks, chunks
-
-        vchunks = range(0, shape[0], v_chunk_size)
-        hchunks = range(0, shape[1], h_chunk_size)
+        chunks = normalize_chunks(chunks, shape, dtype=self.values.dtype)
```

Comment:
Is there no input dask array to take the chunk size from or to calculate the chunk size from (ex. multiply by a factor)?

Reply:
I think this is very dependent on the input data/reader. In the cases where the chunk sizes are known and provided here, `normalize_chunks` should not be interfering.

Eg:
```python
In [6]: normalize_chunks(((2, 2, 2, 2, 2), (2, 2, 2, 2, 2)), (10, 10), dtype=np.float32)
Out[6]: ((2, 2, 2, 2, 2), (2, 2, 2, 2, 2))
```

However, using it allows passing more abstract values, like `10MiB` or `auto`.

Reply:
Sounds good